app/src/main/java/mozilla/lockbox/store/ItemDetailStore.kt

```diff
@@ -0,0 +1,52 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ *
```

Comment:
Nit: whitespace

Reply:
oops!
Aaaand it's in the just-merged template, too.